build.cake

```diff
@@ -197,6 +197,27 @@ Task("TestAtlasConnectivity")
     );
 });
 
+Task("TestOcsp")
+    .IsDependentOn("Build")
+    .DoesForEach(
+        GetFiles("./**/MongoDB.Driver.Tests.csproj"),
+        testProject =>
+{
+    DotNetCoreTest(
+        testProject.FullPath,
+        new DotNetCoreTestSettings {
+            NoBuild = true,
+            NoRestore = true,
+            Configuration = configuration,
+
+            ArgumentCustomization = args => args
+                .Append("--filter FullyQualifiedName~OcspIntegrationTests")
+                .Append("-- RunConfiguration.TargetPlatform=x64")
```

Comment:
can we leave or remove space after `--` in these lines?

Reply:
Sorry, I'm not sure I understand?
For line 215, the space is required: See https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test, "Runsettings argument"